File: 프로그래머스/0/120854. 배열 원소의 길이/배열 원소의 길이.js
function solution(strlist) {
    var answer = [strlist.length];
    
    // answer 배열 요소 => strlist 배열의 i번째 문자열 길이
    for (let i = 0; i < strlist.length; i++) {
        answer[i] = strlist[i].length;
    }
    
    return answer;
}

File: 프로그래머스/0/120854. 배열 원소의 길이/배열 원소의 길이.java
import java.util.*;

class Solution {
    public int[] solution(String[] strlist) {
        int[] answer = {};
        List<Integer> list = new ArrayList<Integer>();
        
        // strlist 배열의 i번째 문자열 길이를 리스트에 추가
        for (int i = 0; i < strlist.length; i++) {
            list.add(strlist[i].length());
        }
        
        // 리스트 => 배열
        return answer = list.stream().mapToInt(i -> i).toArray();
    }
}